hyrisecockpit/database_manager/cursor.py

```diff
@@ -99,6 +99,26 @@ def __write_point(self, point: Point) -> None:
         """Write a single point to the database."""
         return self.__write_points([point])
 
+    def create_database(self, database_name) -> None:
+        """Create database with a given name."""
+        self._connection.create_database(database_name)
+
+    def drop_database(self, database_name) -> None:
+        """Drop database with a given name."""
+        self._connection.drop_database(database_name)
+
+    def create_continuous_query(
+        self,
+        query_name: str,
+        query: str,
+        database_name: str,
+        resample_options: Optional[str] = None,
+    ) -> None:
+        """Create continuous query."""
+        self._connection.create_continuous_query(
+            query_name, query, database_name, resample_options
```

Comment:
The `StorageCursor` is specific to one database. We probably shouldn't expose methods to create and drop other databases than the one it was initialized with.

```suggestion
    def create_database(self) -> None:
        """Create database with a given name."""
        self._connection.create_database(self._database)

    def drop_database(self) -> None:
        """Drop database with a given name."""
        self._connection.drop_database(self._database)

    def create_continuous_query(
        self,
        query_name: str,
        query: str,
        resample_options: Optional[str] = None,
    ) -> None:
        """Create continuous query."""
        self._connection.create_continuous_query(
            query_name, query, self._database, resample_options
```

Reply:
Thanks for the suggestion, we could do so